training/tlt/pedestrian_detection/extract_images.py

```diff
@@ -0,0 +1,39 @@
+import os
```

Comment:
We can probably do this faster and simpler using ffmpeg, something like:
ffmpeg -i TownCentreXVID.avi ./frames/%d.png -hide_banner

Reply:
I mean we don't need a python script to do that :)

Reply:
done.
what a great tool, thanks for suggestion!